test_util/marathon.py

```diff
@@ -202,6 +202,88 @@ def _pool_for_marathon_app(app_id):
             raise Exception("Application deployment failed - operation was not "
                             "completed in {} seconds.".format(timeout))
 
+
```

Comment:
Too many blank lines in this file. @kensipe please clean up so we can pass the CodeQuality check
https://teamcity.mesosphere.io/viewLog.html?buildId=474873&buildTypeId=DcosIo_Dcos_CodeQuality